src/combine_esdf/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>combine_esdf</name>
+  <version>0.1.0</version>
+  <description>Combines 2 Nvblox ESDF slices into a single ESDF slice</description>
+  <maintainer email="aaberg333@gmail.com">Alex Berg</maintainer>
+  <license>MIT License</license>
+
+  <test_depend>ament_copyright</test_depend>
+  <test_depend>ament_flake8</test_depend>
+  <test_depend>ament_pep257</test_depend>
+  <test_depend>python3-pytest</test_depend>
+  <test_depend>numpy</test_depend>
```

Comment:
for numpy and scipy change them from "test_depend" to either "depend" or "exec_depend"